packages/components/tag/src/tag.js

```diff
@@ -231,15 +231,39 @@ const Tag = (props) => {
 };
 
 Tag.propTypes = {
+  /**
+   * Indicates color scheme of the tag.
+   */
   type: PropTypes.oneOf(['normal', 'warning']),
+  /**
+   * Styles object spread onto the tag body..
+   */
   styles: PropTypes.shape({
     body: PropTypes.object,
   }).isRequired,
+  /**
+   * Location the tag links to when enabled.
+   */
   linkTo: PropTypes.string,
+  /**
+   * Disable the tag element along with the option to remove it.
+   */
   isDisabled: PropTypes.bool,
+  /**
+   * Called when remove button is clicked.
```

Comment:
```suggestion
   * Called when tag is removed.
```

Reply:
hmm..
the thing is, is that it's the consumer that executes the remove action as a result of clicking the remove button.

The wording you propose, suggests that `onRemove` is called after the UI-kit removes the tag.

Reply:
Ok. Thanks.